### PR TITLE
Fix invalid spacing

### PR DESCRIPTION
**Pull request by lewischeng-ms**

### CODE REVIEW

File: src/Common.Stylecop [hunk 1]
```diff
@@ -61,6 +61,15 @@
         </Rule>
       </Rules>
     </Analyzer>
+    <Analyzer AnalyzerId="Microsoft.StyleCop.CSharp.MaintainabilityRules">
+      <Rules>
+        <Rule Name="FileMayOnlyContainASingleClass">
```
Comment (congysu) [inline src/Common.Stylecop]: Is this to suppress SA1402?

Comment (lewischeng-ms) [inline src/Common.Stylecop]: Yes, there are many cases in RESTier to do so. Shall we follow this to separate those files to contain only one class per file?

Comment (congysu) [inline src/Common.Stylecop]: Ok to me to suppress it for now.  Thanks.

File: src/Microsoft.Restier.WebApi/ODataDomainController.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ protected ODataDomainController()
         }
 
         /// <summary>
-        /// Gets the <see cref="IDomain"/> interface of the domain of the <see cref="ODataDomainController"/>.
```
Comment (congysu) [inline src/Microsoft.Restier.WebApi/ODataDomainController.cs]: What is the issue with `<see cref="*`?

Comment (lewischeng-ms) [inline src/Microsoft.Restier.WebApi/ODataDomainController.cs]: @karataliu suggested that "Gets the interface of ..." is not proper so change to this statement.
P.S., this comment was originally not present and added by me several days ago. So it's OK to change the way expressing.

**Pull request merged**